pick.lic

```diff
@@ -112,19 +112,28 @@ class LockPicker
   def setup
     arg_definitions = [
       [
-        { name: 'pets', regex: /pets/, optional: true, description: 'Disarm boxes and place them in the pet box container' },
-        { name: 'count', regex: /\d+/, optional: true, description: 'How many pet boxes to make' },
-        { name: 'refill', regex: /refill/, optional: true, description: 'Refills your lockpick ring' }
+        { name: 'pets', regex: /pets/, optional: true, description: 'Disarm boxes and place them in the pet box container.' },
+        { name: 'count', regex: /\d+/, optional: true, description: 'How many pet boxes to make.' },
+        { name: 'force', regex: /force/, optional: true, description: 'Force making the pet box count.' },
+        { name: 'refill', regex: /refill/, optional: true, description: 'Refills your lockpick ring.' }
       ]
     ]
 
     args = parse_args(arg_definitions, true)
 
     @make_pets = args.pets
     @pet_goal = args.count.to_i
+    force = args.force
     @pet_count = 0
 
     @settings = get_settings(args.flex)
+
+    if @make_pets and not force
+      box_count = get_boxes(@settings.picking_box_source).count
+      @pet_goal = @settings.pet_boxes_on_hand - box_count
+      exit unless @pet_goal > 0
```

Comment:
💡 I prefer to use 'if' when possible. In this case, that would mean inverting the conditional, something like `exit if @pet_goal <= 0` (or possibly `exit if @pet_goal.zero?`)